pom.xml

```diff
@@ -29,6 +29,10 @@
     <hamcrest.version>1.3</hamcrest.version>
     <maven-shade-plugin.version>3.0.0</maven-shade-plugin.version>
     <relocation-prefix>com.hotels.hcommon.ssh</relocation-prefix>
+    <guava.version>20.0</guava.version>
+    <hivernate-validator.version>5.4.2.Final</hivernate-validator.version>
+    <javax-el.version>3.0.1-b08</javax-el.version>
+    <spring-webmvc.version>4.3.19.RELEASE</spring-webmvc.version>
```

Comment:
Are you sure we need this? I think this is indirectly pulling in spring-context to get `LocalValidatorFactoryBean` for you but you can probably get it more directly and not have all this MVC stuff required on the classpath.